features/page-breaks.yml

```
@@ -1,11 +1,13 @@
 name: Page breaks
-description: The `break-after`, `break-before`, `break-inside` CSS properties (along with `page-break-` aliases) control where printed pages start and end. Also known as pagination or page breaking.
+description: The `break-after`, `break-before`, `break-inside` CSS properties control where printed pages start and end. Also known as pagination or page breaking.
```

I think we need to take on the description style used for `column-breaks`, which rely on the same properties. Something like this:

```suggestion
description: In printed page layouts, the `break-after`, `break-before`, `break-inside` CSS properties control where printed pages start and end. Also known as pagination or page breaking.
```

That said, the column breaks feature introduces it like this:

> In columnar layouts (created by the `columns` or `column-count` CSS properties), the `break-after`, `break-before`, `break-inside` properties…

I chose not to highlight what creates a print layout (the browser's print dialog), since I _think_ it's kinda obvious, but I wasn't certain.

---

I like your suggested description.